scripts/exgdas_global_aero_analysis_finalize.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+# exgdas_global_aero_analysis_finalize.py
+# This script creates an AerosolAnalysis class
+# and runs the configure and finalize methods
+# which perform post-processing and clean up activities
+# for a global aerosol variational analysis
+import pygfs.task.aero_analysis
+import logging
+import os
+
+# set up logger
```

Comment:
```suggestion
# set up logger TODO: use pygw.Logger
```

Reply:
can you please add `TODO: ` elsewhere too so we can see it.
Better yet, open an issue and self assign, because otherwise we forget.